### DISCUSSION TITLE
Haptic Feedback via Taptic or HD Rumble

### DISCUSSION THREAD

**Discussion by aamott** [inputlabs/organization#30] [Ideas & feature request]
Loving the project going on here. One thing I've been amazed at for years was when a friend showed me a game (I think on the ouya) where you guess the number of balls in a digital "box" by tipping the controller around and feeling how they bounce. The Switch has it now and calls it Ball Count. It uses their HD Rumble. The iPhone's "click" of the home button and all vibrations are made using the Taptic Feedback system. 
Just an idea, what is haptic feedback was implemented with a simple speaker driver and an iPhone taptic feedback unit or JoyCon HD Rumble unit? The advantage is that since it's driven like a speaker, you get more flexibility. Taps, vibrations, and different frequencies and intensities, where a normal motor can only increase its intensity at the same time as its frequency. 
Here's an example where someone [stuck an iPhone 7 taptic feedback unit into an old ipod](https://youtu.be/GZTTusuHpm4). They can be had for about $3-4 including shipping. 
Might be excessive, but I thought it was worth a post.

**Comment by marcos-diaz** [comment]
Hi @aamott,

Yes there are many solutions available, from coin vibrators that are surprisingly strong and responsive nowadays, to different kind of haptic modules.

Nothing is decided yet, we will experiment with these later, and depending on testers feedback, price and other factors we will decide what to include in the reference design. 

Thanks for the suggestion!
